src/run/troubleshoot/analysis/SSHAnalysis.ts

```diff
@@ -0,0 +1,46 @@
+import { existsSync } from "fs";
+import BaseAnalysis from "./AbstractBaseAnalysis.js";
+import AnalysisOutcome from "./AnalysisOutcome.js";
+import { AnalysisIssue, TroubleshootAnalysisTaskContext } from "./AnalysisTask.js";
+
+const ANALYSIS_HEADLINE = "Checks if SSH Keys are present in the local directory";
+
+const SUGGESTIONS = [
+    "Run: 'bin/init' command in the local project directory to initialise SSH keys."
+];
+
+const DOCUMENTATION_LINKS = [
+    "troubleshooting-remedies/correctly-set-aws-environment-variables.md"
```

Comment:
I cannot see any mention of ssh at this doc link.

Reply:
Was going to get rid of that. Thanks.